### PR TITLE
feature/fw Update register periodic time min and CAN ID

### PR DESCRIPTION
**Pull request by RobertoAldea**
Added final values of kpis for control

### CODE REVIEW

File: firmware/project_config/EPC_CONF/epc_conf.c [hunk 1]
```diff
@@ -25,24 +25,24 @@
 #define _MIN_HS_VOLT       5300  // V
 #define _MAX_LS_VOLT       5100  // V
 #define _MIN_LS_VOLT        400  // V
-#define _MAX_LS_CURR      15500  // A
-#define _MIN_LS_CURR     -15500  // A
+#define _MAX_LS_CURR      10000  // A 15500  //PRUEBAS 10000
```
Comment (pastorpflores) [inline firmware/project_config/EPC_CONF/epc_conf.c:28]: Please delete not needed comments

File: firmware/project_config/EPC_CONF/epc_conf.c [hunk 2]
```diff
@@ -124,8 +124,8 @@ const MID_REG_meas_property_s EPC_CONF_MEAS_max_value = {
 };
 
 const MID_REG_meas_property_s EPC_CONF_MEAS_factors = {
-	5000,	//lsVolt max-min
-	33000,	//lsCurr max-min
+	6000,	//lsVolt max-min 6000	// PRUEBAS 5000
```
Comment (pastorpflores) [inline firmware/project_config/EPC_CONF/epc_conf.c:127]: Please delete not needed comments

File: firmware/project_config/EPC_CONF/epc_conf.c [hunk 3]
```diff
@@ -136,7 +136,7 @@ const MID_REG_meas_property_s EPC_CONF_MEAS_factors = {
 
 const MID_REG_meas_property_s EPC_CONF_MEAS_offset = {
 	0,	//lsVolt
-	-16500,	//lsCurr
+	-16500,	//lsCurr -16500 //PRUEBAS -11000
```
Comment (pastorpflores) [inline firmware/project_config/EPC_CONF/epc_conf.c:139]: Idem

**Review by pastorpflores — CHANGES_REQUESTED** [review]

File: firmware/project_config/EPC_CONF/epc_conf.c (hunk 3 above)
Comment (RobertoAldea) [reply firmware/project_config/EPC_CONF/epc_conf.c]: Solved

File: firmware/project_config/EPC_CONF/epc_conf.c (hunk 2 above)
Comment (RobertoAldea) [reply firmware/project_config/EPC_CONF/epc_conf.c]: Solved

File: firmware/project_config/EPC_CONF/epc_conf.c (hunk 1 above)
Comment (RobertoAldea) [reply firmware/project_config/EPC_CONF/epc_conf.c]: Solved

**Review by pastorpflores — APPROVED** [review]

**Review by pastorpflores — APPROVED** [review]

**Pull request merged**